cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/store/DefaultStoreTest.java

```diff
@@ -517,6 +518,98 @@ public void testUpdateChangedApplication() throws ConflictException {
     Assert.assertEquals(FooMapReduceJob.class.getName(), spec.getMapReduce().get("mrJob3").getClassName());
   }
 
+  @Test
+  public void testAddApplicationWithoutMarkingLatest()
+      throws ConflictException {
+    long creationTime = System.currentTimeMillis();
```

Comment:
nit: I dont think you dont need to create this everytime. This can be a class level final value

Reply:
Agreed. But (for now) I would rather keep these values isolated (contained within the context of each test) without adding class level variables.